hazelcast/src/test/java/com/hazelcast/query/impl/predicates/VisitorTestSupport.java

```diff
@@ -124,7 +122,7 @@ private static void assertCompositeEqualPredicate(CompositeEqualPredicate expect
         CompositeEqualPredicate actualEqualPredicate = (CompositeEqualPredicate) actual;
 
         assertEquals(expected.indexName, actualEqualPredicate.indexName);
-        assertArrayEquals(expected.components, actualEqualPredicate.components);
+        assertEquals(expected.components, actualEqualPredicate.components);
```

Comment:
I might miss something, but `assertEquals` for arrays is marked as deprecated and `assertArrayEquals` is the recommended replacement.

Reply:
Fixed in https://github.com/hazelcast/hazelcast/pull/15537/commits/2163cbdd5ac5290a80317aa92679e8ded7520b42
This is an artifact: at some point, during implementation, I changed `getComponents` return type to `List` and changed tests accordingly. Then reverted back to the array, but forgot to revert tests.